### PR TITLE
Add a "get started" page for new contributors

### PR DESCRIPTION
**Pull request by pepicrft**
I'm adding a new page that guides developers through the firs steps they should take to get the project working locally.
I've taken the opportunity to move the tutorial docs under `docs/tutorial`, and disable auto-generation on the sidebar. A manually-configured sidebar allows us to indicate the order in which documents are presented.

### CODE REVIEW

File: packages/docs/docs/contributors/get-started.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Get started
+
+Contributions bring a diversity of ideas to the project,
+which enriches the project.
```
Comment (android10) [inline packages/docs/docs/contributors/get-started.md:4]: is the github editor cutting off the document ruining the format?

File: packages/docs/docs/contributors/get-started.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Because of that,
+we design the project and the documentation to ease contributions.
+
+A contribution can come in many shapes:
+[GitHub issues](https://github.com/gestaltjs/gestalt/issues) reporting bugs or sharing ideas,
+community support,
+framework evangelization,
+or code contributions as [pull requests](https://en.wikipedia.org/wiki/Distributed_version_control#Pull_requests).
```
Comment (android10) [inline packages/docs/docs/contributors/get-started.md]: use a list and hightlight them:

 - **One**
 - **Two**

**Review by android10 — APPROVED** [review]
A couple of opinionated points and then +1

File: packages/docs/docs/contributors/get-started.md (hunk 1 above)
Comment (pepicrft) [reply packages/docs/docs/contributors/get-started.md:4]: I'm following [semantic line break](https://sembr.org/) to make units of thought more distinguishable and easy to work with.

**Pull request merged**